src/Internal/Payment/PaymentContext.php

```diff
@@ -233,6 +264,7 @@ public function get_customer_id(): ?string {
 	 * @param mixed  $value Value to store.
 	 */
 	private function set( string $key, $value ) {
+		$this->changes[]    = new Change( $key, $this->get( $key ) ?? '', $value );
```

Comment:
1. Let's only log _changes_. Presently, everything is only set once, but in the future, we might do the same thing multiple times (ex. extract order data), and we should only log the things that actually changed.
2. `get` would return `null`, let's keep it `null`. In various cases, empty strings are a different thing.